charts/argo/templates/workflow-controller-deployment.yaml

```diff
@@ -61,6 +61,9 @@ spec:
               fieldRef:
                 apiVersion: v1
                 fieldPath: metadata.namespace
+          {{- with .Values.controller.env }}
```

Comment:
Normally `extraEnv` is used for additional env vars in a container.

Reply:
Done.